src/commands/createAccountsService.js

```
@@ -0,0 +1,74 @@
+export class CreateAccountsService {
+  constructor(client = null) {
+    this.client = client || new STSClient();
+  }
+
+  async handleCreateAccountsCommand(options) {
+    config.environment = 'global';
```

ESLint

[error] 'config' is not defined.

(no-undef)